### PR TITLE
added support for other promise libraries

### PR DESCRIPTION
**Pull request by oliverlorenz**
Hi! 

I really love your library! I use it every day. But always have to wrap it because I need some functions from the bluebird library which is a kind of superset of the standard Promise implementation.

With my change its possible (not necessary) to inject any promise library. I hope you will like it and it will be added to your project.

Greetings from Hamburg, Germany
Oliver

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,14 +10,15 @@
     "url": "https://github.com/achingbrain/freeport-promise"
   },
   "scripts": {
-    "test": "NODE_ENV=test mocha",
+    "test": "NODE_ENV=test; mocha",
```
Comment (achingbrain) [inline package.json]: For my own sanity, why is the semi colon necessary?

**Comment by achingbrain** [comment]
This is great, btw..

File: package.json (hunk 1 above)
Comment (oliverlorenz) [reply package.json]: You're right, its not necessary. Only a habit ;) I remove it, one minute :)

**Comment by achingbrain** [comment]
Published as `v1.1.0`.  Thanks!

**Comment by oliverlorenz** [comment]
Cool! Thank you! :)

**Pull request merged**